### DISCUSSION TITLE
How to create 404.blade.php with default filamentphp design?

### DISCUSSION THREAD

**Discussion by PetroGromovo** [filamentphp/filament#6573] [Help]
My laravel/filamentphp site has resources/views/errors/404.blade.php with structure like :

    @extends('layouts.frontpage')
    
    @section('content')
    
        <div >
    
        </div>
    
    @endsection

where file consisit of html tags and php-blade commands.

How can I create such file with filamentphp, based on standart layout/design and possibility to show custom messages?

I created manually file resources/views/errors/404.blade.php - it is rendered, but it has no default filamentphp design. How can I implement it ?

    "filament/filament": "^2.17.17",
    "filament/forms": "^2.17.17",
    "laravel/framework": "^9.52.4",

Thanks in advance!

**Comment by ArtMin96** [comment]
```blade
<x-filament::page>
    //...
</x-filament::page>
```

**Reply by PetroGromovo** [reply]
I created a file resources/views/errors/404.blade.php with content :
```

<x-filament::page>

    <h1 class="filament-header-heading text-2xl font-bold tracking-tight">
        You entered invalid url !
    </h1>

    <p class="filament-header-subheading max-w-2xl tracking-tight text-gray-500">
        <a class="cursor-pointer" href="{{ url('/') }}">Home</a>
    </p>

</x-filament::page>

```
and got error on invalid page : https://prnt.sc/lBC9OLMJCHVP
